masterlist.yaml

```diff
@@ -5909,6 +5909,77 @@ plugins:
         util: 'SSEEdit v3.2.2'
       - crc: 0x98AD850B # vLITE
         util: 'SSEEdit v3.2.2'
+  - name: 'Winterhold Restored.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12811/'
+        name: 'Winterhold Restored on Nexus Mods'
+    group: 'Add-ons & Expansions'
+    inc:
+      - 'ExpandedWinterholdRuins.esp'
+      - 'Less desolate Winterhold.esp'
+      - 'QaxeWinterholdRebuild.esp'
+      - 'sm_winterholdtemple.esp'
+      - 'The Great City of Winterhold.esp'
+      - 'Winterholdtruecapitalruins.esp'
+      - 'Winterholdtruecapitalruinsjkversion.esp'
+      - 'WinterholdDestruction.esm'
+      - 'Winterhold Fully Restored.esp'
+    after:
+      - 'Bring Out Your Dead.esp'
+      - 'CFTO.esp'
+      - 'Clockwork.esp'
+      - 'CollegeOfWinterholdImmersive.esp'
+      - 'Cutting Room Floor.esp'
+      - 'Immersive Citizens - AI Overhaul.esp'
+      - 'MagicalCollegeofWinterhold.esp'
+      - 'Skyrim Better Roads - Winterhold.esp'
+      - 'sm_winterholdgate.esp'
+      - 'Thunderchild - Epic Shout Package.esp'
+      - 'TKAA.esp'
+    msg:
+      - type: warn
+        condition: 'active("sm_jobasha_bookshop.esp")'
+        content: 'Jobasha''s Books SE is only compatible with its Winterhold Restored version.'
+        subs:
+          - 'Jobasha''s Books SE'
+          - '[Jobasha''s Books and Winterhold Restored Patch](https://www.nexusmods.com/skyrimspecialedition/mods/13285/)'
```

Comment:
I would replace this with an incompatible message which only displays if the patch is not active.

```yaml
    inc:
      - 'ExpandedWinterholdRuins.esp'
      - 'Less desolate Winterhold.esp'
      - 'QaxeWinterholdRebuild.esp'
      - name: 'sm_jobasha_bookshop.esp'
        display: '[ Jobasha''s Books SE ](https://www.nexusmods.com/skyrimspecialedition/mods/13285/)'
        condition: 'active("sm_jobasha_bookshop.esp") and not active("sm_jobasha_bookshop.esp")'

```

Reply:
This is actually not a patch but a replacement version. You only use one of the plugins. Both are full versions of the mod.